client/main.lua

```diff
@@ -252,10 +252,8 @@ RegisterNetEvent('qb-vehiclekeys:client:GiveKeys', function(id)
     end
 end)
 
-RegisterNetEvent('QBCore:Client:VehicleInfo', function(data)
-    if data.event == 'Entering' then
-        robKeyLoop()
-    end
+RegisterNetEvent('QBCore:Client:EnteringVehicle', function()
```

Comment:
The event `QBCore:Client:EnteringVehicle` no longer exists. The event handler that existed already is fine.